backend/src/main/kotlin/app/ehrenamtskarte/backend/common/webservice/GraphQLHandler.kt

```diff
@@ -69,15 +73,40 @@ class GraphQLHandler(
         return result
     }
 
+    private fun getJwtTokenFromHeader(context: Context): String? {
+        val header = context.header("Authorization") ?: return null
+        val split = header.split(" ")
+        return if (split.size != 2 || split[0] != "Bearer") null else split[1]
+    }
```

Comment:
As this involves cross-origin headers, we have to be a little bit careful... @maxammann Do you remember what we need to take into account here s.t. browsers don't block fetches with headers from `druckerei.*` to `api.*`? I think we solved this problem in some prior project...

Reply:
Something like that: https://www.sjoerdlangkemper.nl/2018/09/12/authorization-header-and-cors/#by-the-application ?

Reply:
Probably, yes :+1: I thought, it might be harder than that though... But yeah :sweat_smile:

Reply:
@Schedulaar I would add that when implementing the login on frontend side though, so that I can test it properly.